src/core_plugins/metrics/public/visualizations/components/horizontal_legend.js

```diff
@@ -0,0 +1,36 @@
+import React, { PropTypes } from 'react';
+import _ from 'lodash';
+import createLegendSeries from '../lib/create_legend_series';
+
+function HorizontalLegend(props) {
```

Comment:
ehm ... maybe i don't understand this correctly ... but all other components are using class syntax. dont you want to create a base class Legend and then extend it with HorizontalLegend and VerticalLegend ?